harness/determined/ipc.py

```diff
@@ -196,6 +212,15 @@ def send(self, obj: Any) -> None:
         self._send_serial += 1
         self._push_socket.send_pyobj(message)
 
+    def send_priority_message(self, obj: Any) -> None:
```

Comment:
We would not need this if we transitioned to using normal TCP connections for the BroadcastServer and BroadcastClient.  We would just see the connection broken when the broken socket connection when a child died and we would call it a day.

AFAIK that's there's no reason we can't transition BroadcastServer to TCP right now; we could use the same exact ports as we do for the ZMQ server